onnx2kerastl/reshape_layers.py

```diff
@@ -291,17 +291,20 @@ def convert_unsqueeze(node, params, layers, lambda_func, node_name, keras_name):
         else:
             raise AttributeError('Number of inputs is not equal 1 for unsqueeze layer')
 
+    if len(np.unique(params['axes'])) < len(params['axes']):
+        raise AttributeError(f"The specified axes contains duplicates values: {params['axes']}")
```

Comment:
Did we see duplication of axes in a recent onnx model?

Reply:
no, but in onnx docs it's mention that this is not allowed so I added a check